web-console/3-project-details/jobs.md

```diff
@@ -0,0 +1,59 @@
+
+
+# Jobs
+
+- A Job creates one or more pods which run to completion, i.e. they don't run continuously like deployments.
+- A Cron Job extends a regular Job by allowing users to schedule when a job will run.
+- This documentation uses "Job" to refer to both Jobs and Cron Jobs, except when Cron Jobs are called out specifically.
+
+## Overview
+- Jobs and Cron Jobs would be listed under a "Jobs" heading in the Application and Resource views of the Overview.
+- The heading over individual Job names would specify if it is a Job or Cron Job.
+- Jobs (and Cron Jobs) that have an app label should be grouped with that application in the Application view.
+
+### Collapsed State
+![jobs overview collapsed](img/jobs-overview-collapsed.png)
+- Jobs are listed on the Overview page similarly to deployments.
+- Currently running jobs have the pod donut donut.
+- For Jobs that are not running, the pod donut is replaced by a "Start Job" button which would initiate a job run.
```

Comment:
Do you have an example of what the job looks like once you click start? Does a grey donut appear at first? Just remembering that we have issues around the lack of user feedback when clicking "Start Pipeline" today for example so want to avoid creating additional issues like that where we don't give user feedback on an action.

Reply:
The start button should be replaced by some kind of donut as soon as you click the button. Not sure if that would be an empty dount, or what its state would be, but I would imagine that it would reflect the state just like with anything else that has pods.

Reply:
I'll add a comment about that.